packages/hint/docs/user-guide/index.md

```diff
@@ -26,7 +26,7 @@ best practices that may be applied to your site, complete the following steps.
    uses `npx` to run the `npm` package without installing it.
 
    ```bash
-   npx hint https://example.com
+   npx hint https://example.com // or relative path e.g. './path/to/my.html'
```

Comment:
What do you think about breaking these apart as two complete commands? I think that might help for those that want to copy-paste a local file example with minimal required changes. E.g.

----

```bash
npx hint https://example.com
```

or

```bash
npx hint ./path/to/my.html
```

Reply:
sure, it's nicer this way